factor/email/db/install.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<XMLDB PATH="admin/tool/mfa/factor/email/db" VERSION="20191106" COMMENT="XMLDB file for Moodle tool/mfa plugin - email factor"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="../../../../lib/xmldb/xmldb.xsd"
+>
+    <TABLES>
+        <TABLE NAME="tool_mfa_factor_email" COMMENT="E-Mail Factor data for MFA plugin">
```

Comment:
See above, I don't think we need this table

Reply:
Corrected